### DISCUSSION TITLE
Welcome to vosk-api-jetson Discussions!

### DISCUSSION THREAD

**Discussion by sskorol** [sskorol/vosk-api-jetson#1] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by DocksDocks** [comment]
**Using python with the docker build? #29**

Hello @sskorol 
Can you delete the other discussion that I've created, I just noticed that you already had created one for questions...

My question:

I created a file called mytest.sh and basically used the sample of test.sh, but put the dependencies of my code...
```
#!/bin/bash

TAG=${1:-latest}

echo "TAG=$TAG" > .env \
  && docker-compose up -d \
  && python3 -m venv .venv \
  && source .venv/bin/activate \
  && sudo apt install espeak python3-pyaudio \
  && pip3 install pip --upgrade \
  && pip3 install pypi-json vosk  \
  && sleep 10 \
  && python3 mytest.py \
  && deactivate \
  && docker-compose down
```
I want to make a code to recognize my voice,
My code is the one below, in other platforms it works just fine:
```
#!/usr/bin/env python3

from vosk import Model, KaldiRecognizer
import pyaudio
import json

# Speech Recognition
model = Model('model')
rec = KaldiRecognizer(model, 16000)

p = pyaudio.PyAudio()
stream = p.open(format=pyaudio.paInt16, channels=1, rate=16000, input=True, frames_per_buffer=8192)
stream.start_stream()

# Loop speech recognizing
print('Recognizing...')
while True:
	data = stream.read(4096)
	if rec.AcceptWaveform(data):
		result = rec.Result()
		result = json.loads(result)
		if result is not None:
			text = result['text']
			# Text result
			print('Text: {}'.format(text))
			break
print('End')
```

The problem is when downloading the 'pip3 install vosk' it results in this error: 
```
OSError: cannot load library '/home/jetson/projects/vosk-api-gpu/.venv/lib/python3.6/site-packages/vosk/libvosk.so': /home/jetson/projects/vosk-api-gpu/.venv/lib/python3.6/site-packages/vosk/libvosk.so: undefined symbol: _ZNSt7__cxx1119basic_ostringstreamIcSt11char_traitsIcESaIcEEC1Ev
 ```
The problem is that I probably should not use vosk library, but how else can i execute the code like this with the build and get the same result of in real time speech recognition?
